frontend/src/components/ScanOption.tsx

```
@@ -0,0 +1,139 @@
+        return (
+          <div className="space-y-2">
+            <Label htmlFor={name} className="text-sm text-muted-foreground">
+              {displayName}
+            </Label>
+            <Input
+              id={name}
+              type="number"
+              value={value as number}
+              onChange={(e) => {
+                const val = parseInt(e.target.value);
+                if (!isNaN(val)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)